### PR TITLE
Standards Import: use curriculum.code.org instead of codecurricula 

### PR DESCRIPTION
**Pull request by Erin007**
Follow up to #33123. 

I tried to import the standards on production and still hit this error: 
Screen Shot 2020-02-13 at 4 52 44 PM

Because I allowed https: codecurricula  NOT http: codecurricula 🤦‍♀️

Update: Switching to curriculum.code.org.  I think this would've worked if I'd changed the ajax url to be https for codecurricula, but curriculum.code.org makes more sense regardless since it's the published up-to-date content that we'd want to be referencing.

### CODE REVIEW

**Comment by bethanyaconnor** [comment]
Why are we using http for codecurricula?

**Comment by Erin007** [comment]
> Why are we using http for codecurricula?

I don't know, but we are. Are you concerned that it makes codecurricula more vulnerable, which in turn would make allowing fetching data from it a vulnerability for code studio?

**Comment by bethanyaconnor** [comment]
I don't know the details of this feature so I don't know what data is passed in the request to codecurricula. I don't think this should block this PR though

**Review by bethanyaconnor — APPROVED** [review]
Approving to unblock.

**Comment by Erin007** [comment]
This is an example of the information we're requesting from codecurricula: https://user-images.githubusercontent.com/12300669/72313225-85920400-363f-11ea-81c0-8654a967dedb.gif

**Comment by wjordan** [comment]
Not sure if this applies here, but just a warning that serving ['Mixed Content'](https://developers.google.com/web/fundamentals/security/prevent-mixed-content/what-is-mixed-content) (fetching HTTP content from an HTTPS site) is not only a potential security vulnerability for the page (depending on how the fetched HTTP data is used/displayed) as mentioned above, but it will also probably break most modern browsers that block mixed content by default as part of their own security model (see [Chrome example](https://developers.google.com/web/fundamentals/security/prevent-mixed-content/what-is-mixed-content#an_xmlhttprequest_example)).

At the very least, I'd assume this won't work until we've tested it works over HTTPS on several browsers in our matrix.

**Comment by Erin007** [comment]
Interesting. Thanks, @wjordan! 

I think the better solution is to ask for the metadata from curriculum.code.org since it uses https. But I don't know how to get the changes from codecurricula published to curriculum.code.org (eg. https://curriculum.code.org/metadata/coursea-2019/standards.json says the requested page wasn't found, but http://www.codecurricula.com/metadata/coursea-2019/standards.json has the metadata I want). Stay tuned...

File: apps/src/sites/studio/pages/admin_standards/index.js
Comment (Erin007) [inline apps/src/sites/studio/pages/admin_standards/index.js:5]: @bethanyaconnor codecurricula isn't really an http it's https, but I was asking for http and it still worked, so I don't actually think there's a larger issue there with curriculum builder. Sorry for the confusion!

**Pull request merged**
